src/pages/questionPage.js

```diff
@@ -24,6 +24,18 @@ export const initQuestionPage = () => {
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
```

Comment:
It would be more meaningful if we could rename this `key` keyword. Otherwise, it gets hard while reading and understanding the key below. Maybe something like `answerLetter`?